### ISSUE TITLE
How to install generate_tiles.py

### ISSUE THREAD

**Issue by Ircama** [StyXman/elevation#3]
As your [generate_tiles.py](https://github.com/StyXman/elevation/blob/master/generate_tiles.py) version [needs python-3.6 ](https://github.com/StyXman/elevation/blob/master/generate_tiles.py#L1) and related prerequisites, I have put here a possible installation procedure including [python 3.6](https://askubuntu.com/questions/865554/how-do-i-install-python-3-6-using-apt-get/865569#865569):

```shell
# Download generate_tiles.py
wget https://raw.githubusercontent.com/StyXman/elevation/master/generate_tiles.py
wget https://raw.githubusercontent.com/StyXman/elevation/master/map_utils.py

# Install python-3.6
sudo apt-get install build-essential python-dev python-setuptools libboost-python-dev libboost-thread-dev -y
sudo add-apt-repository -y ppa:jonathonf/python-3.6
sudo apt-get update
sudo apt-get install -y python3.6
sudo apt-get install -y python3.6-dev

# Install python-3.6 prerequisites
wget https://bootstrap.pypa.io/get-pip.py
sudo python3.6 get-pip.py
sudo python3.6 -m pip install --upgrade pip setuptools wheel
sudo python3.6 -m pip install shapely

# Install python-3.6 python-mapnik
git clone -b v3.0.x https://github.com/mapnik/python-mapnik
cd python-mapnik
BOOST_PYTHON_LIB=boost_python-py35 python3.6 setup.py install
cd ..
```

Additional notes:

- *generate_tiles.py* requires to have the XML file in the current directory.
- When running *generate_tiles.py*, you might get the following error if python3.6 is not installed: `/usr/bin/env: ‘python3.6’: No such file or directory`
- Tou might get the following error if map_utils.py is not downloaded: `ModuleNotFoundError: No module named 'map_utils'`

**Comment by StyXman** [comment]
> generate_tiles.py requires to have the XML file in the current directory

No, you can point it to any input file, relative or absolute paths. Files declared in the style have to be either absolute or relative to that file.

**Comment by StyXman** [comment]
Yes, I should improve this, maybe even fork this into its own project. I'll have some time this week, I'll see how feasible that is.

**Comment by StyXman** [comment]
In fact `py-3.6` is not needed, but `py-3` is, so I just change the `#!` line and that should simplify the install a lot.

**Comment by StyXman** [comment]
So now instructions should be more like:

* `sudo apt-get install python3-shapely python3-mapnik`
* `wget https://raw.githubusercontent.com/StyXman/elevation/master/generate_tiles.py`
* `wget https://raw.githubusercontent.com/StyXman/elevation/master/map_utils.py`

Etc.
